server/api/subteam/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from api.team.models import Team
+
+
+class SubTeam(models.Model):
+    subteam_id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=254)
+    team_id = models.ForeignKey(Team, on_delete=models.CASCADE, related_name='teamOf', default="")
```

Comment:
you dont need related_name or default

Reply:
yup have deleted unnecessary arguments and migrated